docs/contributing/issues.md

```diff
@@ -0,0 +1,79 @@
+
+## Searching and Voting
+
+Before opening an issue, please [search the existing issues](https://github.com/openbullet/OpenBullet2/issues?q=is%3Aissue) to see if a similar problem or feature request has been reported. Duplicate issues clutter the repository and should be avoided.
+
+If you do find an issue that matches, or closely matches, your issue, please make use of the :+1: reaction to confirm the issue also affects you or that you support the feature request. If you wish, add a comment as well describing your version of the issue or feature use case.
+
+If the existing issue is closed, please read through it to see if the accepted workaround(s) apply to your case. If not, leave a comment and the issue will be reopened. Note that, since PRs go into `dev` first but releases are built from `master`, an issue's fix won't be immediately available in the official sources, but will be included in the next release.
+
+## Opening an Issue
+
+Once you're ready to open an issue, please [see this page](https://github.com/openbullet/OpenBullet2/issues/new/choose)!
+
+### Reporting Bugs
+
+When writing a bug issue, please ensure you capture as much relevant detail as possible - this is very important to assist in troubleshooting and triaging/investigating the issue. Some useful elements include:
+
+- How you installed OpenBullet2 (upgrade or fresh install)
+
+- What platform and operating system you are using (Debian, Windows, Docker, etc...)
+
+- What you were doing that caused the issue to appear
+
+- Any relevant log output
+
+- Any non-standard configurations you use
+
+Bugs should be tagged with `[bug]` at the beginning of their title. This will later be removed by the OpenBullet2 team when assigning labels. To assist in triaging, if you know which other [label(s)](/docs/general/contributing/issues#issue-labels) should be applied to your issue, please add them after the `[bug]` label.
+
+Bugs should be reproduceable. That is, you should be able to have determined through troubleshooting how to replicate the issue. While one-time bugs should not be ignored, if they're difficult or impossible to reproduce, it's likely very hard to fix them. Please attempt to reproduce the bug before filing the issue and include the smallest test case you can to demonstrate it.
+
+If you ever need assistance for troubleshooting or opening an issue, please [contact the community](https://discourse.openbullet.dev/) and we'll try to help you out!
+
+## Issue Labels
+
+OpenBullet features a number of issue labels to assist in triaging and managing issues. Users cannot assign these themselves due to GitHub's permissions, but they will be added by a team member during triaging.
+
+### Categories
+
+These labels are broad categories for which part of the codebase is affected.
```

Comment:
This needs to be expanded with all the significant labels

Reply:
I think the labels are fine now, what do you think?

Reply:
If you are satisfied with it, I'm fine too!

Reply:
Yes I'm not gonna add more labels, please describe them in this list, thanks!

Reply:
I think you should add this label for pull request:
https://github.com/Gauvino/openbullet2-docs/blob/a563f53a580de61d9f65ae888794447b970f5f6c/docs/contributing/issues.md?plain=1#L94

And this one too:
https://github.com/Gauvino/openbullet2-docs/blob/a563f53a580de61d9f65ae888794447b970f5f6c/docs/contributing/issues.md?plain=1#L79